Remotion/Web/ClientScript/Scripts/SmartNavigation.ts

```diff
@@ -153,46 +153,45 @@ class SmartFocus
     var activeElementID = data;
     if (! StringUtility.IsNullOrEmpty (activeElementID))
     {
-      var activeElement = $('#' + activeElementID);
+      var activeElement = document.getElementById(activeElementID);
       if (activeElement)
       {
-        var isEnabledFilter = function (this: HTMLElement)
+        function isEnabledFilter(el: HTMLElement)
         {
-          var _this = $ (this);
-          if (_this.is ('button'))
+          if (el.tagName === 'BUTTON')
             return true;
-          if (_this.is ('input[type=button]'))
+          if (el.tagName === 'INPUT' && (el as HTMLInputElement).type.toLowerCase() === "button")
             return true;
-          if (_this.is ('input[type=submit]'))
+          if (el.tagName === 'INPUT' && (el as HTMLInputElement).type.toLowerCase() === "submit")
             return true;
-          if (_this.is ('a'))
+          if (el.tagName === 'A')
             return true;
-          if (!_this.is ('a') && _this.is (':enabled') && !_this.is ('input[type=hidden]'))
+          if ((el as HTMLInputElement).disabled === false && !(el.tagName === 'INPUT' && (el as HTMLInputElement).type.toLowerCase() === "hidden"))
```

Comment:
```suggestion
          if (el.tagName !== 'A' && (el as HTMLInputElement).disabled === false && !(el.tagName === 'INPUT' && (el as HTMLInputElement).type.toLowerCase() === "hidden"))
```

I know it's redundant, the reasoning is to make the individual cases easier to reason over -- you have everything you need to understand a single IF in one line instead of needing to check what other exit conditions there hav been.